charts/rancher-wins-upgrader/templates/daemonset.yaml

```diff
@@ -37,7 +37,7 @@ spec:
         - name: wins-pipe
           mountPath: \\.\pipe\rancher_wins
         - name: wins
-          mountPath: c:/host/etc/rancher/wins
+          mountPath: {{ include "windows_host_path" $ }}etc/rancher/wins
```

Comment:
This could be `winsUpgrader.winsHostPath` from above. Is this intentional?

Reply:
I just kept it the same and substituted the one part of the path that needed to be different. Happy to change it to use the winsHostPath.

Reply:
I think that would be a good change :)